sp/src/game/server/ez2/npc_flyingpredator.cpp

```diff
@@ -169,6 +189,7 @@ BEGIN_DATADESC( CNPC_FlyingPredator )
 	DEFINE_FIELD( m_flDiveBombRollForce, FIELD_FLOAT ),
 
 	DEFINE_KEYFIELD( m_bCanUseFlyNav, FIELD_BOOLEAN, "CanUseFlyNav" ),
+	DEFINE_KEYFIELD( m_bPassiveWhenPerched, FIELD_BOOLEAN, "PassiveWhenPerched" ),
```

Comment:
PassiveWhenPerched seems very similar to ``m_bDormant`` / ``"Dormant"`` in the base predator class. What is the intended function of PassiveWhenPerched? Is it meant to be like Dormant, but not for sounds and smells?

Reply:
I think we need to carefully document the difference between the two, including in the FGD. They are almost identical but they serve different functions.